shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -741,6 +781,16 @@ public static TextDirection fromInt(int value) {
         }
     }
 
+    private class LocalContextAction {
+        LocalContextAction() {
+            resourceId = ++nextContextId;
+        }
+        final int resourceId;
+        int id = -1;
+        String label;
+    }
+    static int nextContextId = 1232132123;
```

Comment:
Maybe add some comment on why `1232132123` is picked and why it's safe to do so. It's also nice to add some asserts to enforce such safety.

Reply:
woops, I should have updated this.  It really just needs to be one bigger than the largest action on Android